### PR TITLE
blackifying all the things

### PR DESCRIPTION
**Pull request by lwasser**
me likey black weeeee!!!

### CODE REVIEW

File: earthpy/clip.py [hunk 1]
```diff
@@ -77,7 +77,7 @@ def clip_line_poly(shp, clip_obj):
 
     # Clip the data - with these data
     clipped = shp_sub.copy()
-    clipped['geometry'] = shp_sub.intersection(poly)
+    clipped["geometry"] = shp_sub.intersection(poly)
```
Comment (lwasser) [inline earthpy/clip.py]: @betatim should't dataframe cols be single quotes? thought that was the rule! maybe i am wrong.

File: earthpy/mask.py [hunk 2]
```diff
@@ -22,18 +22,66 @@
...
+                432,
+                480,
+            ],
             "Medium Cirrus Confidence": [],
-            "High Cirrus Confidence": [834, 836, 840, 848, 864, 880, 898, 900,
-                                       904, 912, 928, 944, 992],
-            "Terrain Occlusion": [1346, 1348, 1350, 1352]
-        }
+            "High Cirrus Confidence": [
+                834,
```
Comment (lwasser) [inline earthpy/mask.py]: this makes me a bit sad given all of the lines but...

File: earthpy/spatial.py [hunk 3]
```diff
@@ -110,34 +113,37 @@ def stack_raster_tifs(band_paths, out_path, arr_out=True):
...
+            with rio.open(out_path, "w", **dest_kwargs) as dest:
                 stack(sources, dest)
             # Read and return array
-            with rio.open(out_path, 'r') as src:
+            with rio.open(out_path, "r") as src:
                 return src.read(), src.profile
         else:
             # Write stacked gtif file
-            with rio.open(out_path, 'w', **dest_kwargs) as dest:
+            with rio.open(out_path, "w", **dest_kwargs) as dest:
```
Comment (lwasser) [inline earthpy/spatial.py]: similarly @betatim i thought that arguments should be single quotes too.

File: earthpy/clip.py (hunk 1 above)
Comment (lwasser) [reply earthpy/clip.py]: ok i get it. they like double quotes every.where. ok fine. :)

**Comment by mbjoseph** [comment]
Heya @lwasser this is looking pretty good! One quick question: I noticed that black wasn't run on `./setup.py` or `docs/conf.py`: is that on purpose? Looking at Tim's PR, I expect that we will be running black from the top-level directory, which means it should hit these two files. 

If you execute `black .` from the top level earthpy directory, does it hit `setup.py` and `docs/conf.py` for you locally? If so, it seems like we could commit those changes to this branch too. After including changes to those files, I think we can merge!!

**Comment by lwasser** [comment]
okdokey! @mbjoseph fixed PLUS i fixed the line length. changed to 79 to match Tim's pr!

**Comment by lwasser** [comment]
how did i miss the SEAL of approval? 😆 !!!

**Pull request merged**